src/scenes/home/faq/question/question.js

```diff
@@ -8,6 +8,7 @@ class Question extends Component {
       <div className={styles.accordionSingle}>
         <input className={styles.accordionSingleHidden} type="checkbox" id={this.props.q} />
         <label className={styles.accordionSingleQuestion} htmlFor={this.props.q}> {this.props.q}</label>
+        { // eslint-disable-next-line react/no-danger react/prop-types: 0) }
```

Comment:
Did you check to see if it still rendered properly w/o errors? I don't know how you can have JS comments right within the JSX.